doc/rst/source/reference/map-projections.rst

```diff
@@ -308,8 +308,8 @@ stereonet can be obtained by using the stereographic projection
 
 .. _-Js:
 
-Stereographic Equal-Angle (**-Js** **-JS**)
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Stereographic equal-angle projection (**-Js** **-JS**)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Comment:
```suggestion
Stereographic equal-angle projection (**-Js** **-JS**)
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

Reply:
The lengths of the underlines are adjusted in commit https://github.com/GenericMappingTools/gmt/pull/8567/commits/2a26c0a431bda505bc1ce1e97096ae0ac86ad5cf.